### DISCUSSION TITLE
Is there a way to configure the key results to reflect matches from more than one RefinableString?

### DISCUSSION THREAD

**Discussion by tedtninety** [microsoft-search/pnp-modern-search#2739] [Q&A] [answered]
I have been tinkering with the GUI every way possible, but I can't get it figured.

I have a PNP Search Result which returns all matches against 'Title' ( RefinableString) that is incurred from a keyword searched.

However, I would also like the results to return any matches from another RefinableString that contains the 'Description' - metadata for the aforementioned "Title'.

Is there a way to configure the key results to reflect  matches from more than one RefinableString? In this instance, the  results would be culled from both the "Title" and 'Description'  RefinableStrings?

**Comment by wobba** [comment, marked as answer]
`RefinableString100:({searchTerms}) OR RefinableString200:({searchTerms})`

**Reply by tedtninety** [reply]
Hi! First off, please forgive my fan-boying. Wow, to me, you are legend! Thank you for all your brilliance!

Thank you for your reply. Apologizes for my newbness, may I please ask where I should place that code?
I searched through the PnP Search Results and the PnP Search Filters  and I can't figure where to modify or append those lines of code.

Thank you again for everything!! :)
ted

**Reply by wobba** [reply]
It's the query template portion of the search result web part properties.
![image](https://user-images.githubusercontent.com/51104/217010070-dd87c39c-0f31-41cd-8314-0c738e063d5a.png)

**Reply by tedtninety** [reply]
Thank you Mikael!!
Going off on what you shared, I had _just_ found it. LOL

Sincerely grateful for you!
Wishing you all the good karma you deserve and more,
ted

**Reply by tedtninety** [reply]
> It's the query template portion of the search result web part properties. ![image](https://user-images.githubusercontent.com/51104/217010070-dd87c39c-0f31-41cd-8314-0c738e063d5a.png)

Hi Mikael. Hope you don't mind me asking, is there a way to make this work on Microsoft Search as well , appending the RefinableStrings after the path?
Thank you.

**Reply by wobba** [reply]
@tedtninety can you explain what you mean, and where?

**Reply by tedtninety** [reply]
@wobba Apologies for the obscurity. 
When in query template, I have two RefinableStrings , like shown above with an OR clause.
**Available Data Sources** is on "**SharePoint Search**". If I toggle **ADS** to "**Microsoft Search**" instead, the two ReinableStrings in the Query Template cease to yield results. It goes blank.
I proceed to re-add the RefinableStrings "Edit Layout Slots" , but still no returns.
What am I doing incorrectly?

Thank you.

**Reply by wobba** [reply]
The provider should not make any difference, depending on what item types you have checked to return using the Graph provider. Tested with driveItems now and worked just fine.

<img width="148" alt="image" src="https://user-images.githubusercontent.com/51104/219013921-29282f19-1797-48eb-9555-de0014d0a529.png">
